code/modules/antagonists/clown_ops/outfits.dm

```diff
@@ -29,3 +29,28 @@
 	command_radio = TRUE
 
 	id_trim = /datum/id_trim/chameleon/operative/clown_leader
+
+/datum/outfit/syndicate/clownop/plasmaman
+	name = "Clown Operative - Plasmaman"
+	uniform = /obj/item/clothing/under/plasmaman/clown
+	head = /obj/item/clothing/head/helmet/space/plasmaman/clown
+	r_hand = /obj/item/tank/internals/plasmaman/belt/full
+	backpack_contents = list(/obj/item/kitchen/knife/combat/survival,
+		/obj/item/dnainjector/clumsymut, //in case you want to be clumsy for the memes
```

Comment:
clown op plasmaman can't give themselves the clumsiness mutation with their dna mutator, so maybe you should just remove it from their outfit

Reply:
Thanks for the catch, @ATH1909